lib/github-package.js

```diff
@@ -360,10 +359,9 @@ export default class GithubPackage {
     };
 
     const items = [
-      this.workspace.getActivePaneItem(),
       this.workspace.getCenter && this.workspace.getCenter().getActivePaneItem(),
     ];
-    const [active, center = {}] = await Promise.all(items.map(fromPaneItem));
+    const [active = {}] = await Promise.all(items.map(fromPaneItem));
```

Comment:
:point_up: This can be simplified now that `items` only contains a single element:

```js
const active = await fromPaneItem(this.workspace.getCenter().getActivePaneItem());
```